extensions/wikia/PortabilityDashboard/SpecialPortabilityDashboardController.php

```diff
@@ -22,18 +22,19 @@ public function index() {
 		$wikiUrlParam = $this->getVal( static::WIKI_URL_QS_PARAM, '' );
 		$isLangFilterSet = !empty( $langFilter );
 		$isWikiUrlParamSet = !empty( $wikiUrlParam );
-		$noResultsInfoMessage = wfMessage(
-			$isLangFilterSet ? 'portability-dashboard-no-results-for-lang-info' : 'portability-dashboard-no-results-for-url-info',
-			PortabilityDashboardModel::WIKIS_LIMIT
-		)->text();
+		$noResultsInfoMessageKey = 'portability-dashboard-no-results-for-url-info';
```

Comment:
It's a matter of personal preference - mine is to put it in 'else' block there:
https://github.com/Wikia/app/pull/11133/commits/9f18b4416d0db93ebd1a34e4feeda4a6538468c4#diff-687fc361036c5b5b06951c4186cbb8f0R36